src/main/kotlin/no/nav/tms/varselbjelle/api/varselRoute.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.tms.varselbjelle.api
+
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import no.nav.tms.varselbjelle.api.notifikasjon.NotifikasjonConsumer
+
+fun Route.varsel(notifikasjonConsumer: NotifikasjonConsumer, varselsideUrl: String) {
+
+    get("rest/varsel/hentsiste") {
+        executeOnUnexpiredTokensOnly {
```

Comment:
Kan fjerne denne wrapperen. Mener dette var for å beskytte seg mot tilfeller der openam-token var gyldig, men oidc-sesjonen var utløpt. Bør fjernes fra dittnav-api på et eller tidspunkt også.